nuntium/templates/nuntium/message/message_in_list.html

```diff
@@ -28,7 +28,10 @@
 	{% endfor %}
 
 	<br />
-	<span class="author_name col-md-4"><small>{% blocktrans with message.author_name as name %}Asked by {{ name }}.{% endblocktrans %}</small></span>
+	<span class="author_name col-md-4"><small>
+    {% url 'all-messages-from-the-same-author-as' slug=message.writeitinstance.slug message_slug=message.slug as other_messages_by %}
+    {% blocktrans with message.author_name as name %}Asked by 
+    <a href="{{other_messages_by}}">{{ name }}</a>.{% endblocktrans %}</small></span>
```

Comment:
I thought so too but apparently the way you define a url to be used in a blocktrans is by previously call the ```{% url as name %}``` filter and then call the blocktrans, according to this [stackoverflow](http://stackoverflow.com/questions/11271002/django-trans-and-url-tags).